Reneiski_Homework1/Reneiski_Homework1/Program.cs

```diff
@@ -1,28 +1,12 @@
-using Reneiski_Homework1.Lecture5;
-using WorkerNamespace;
+﻿using Reneiski_Homework1.Lecture9;
 
 namespace Reneiski_Homework1
 {
-    public class Program
+    public class Program : Lecture9Task
```

Comment:
not a good idea to inherit from Lecture9Task
as you have static methods in Lecture9Task.cs, you can just write Lecture9Task.Task2() in your Main method

Reply:
Fixed